src/software/jetson_nano/services/motor.cpp

```diff
@@ -439,23 +440,31 @@ TbotsProto::MotorStatus MotorService::poll(const TbotsProto::MotorControl& motor
           encoder_calibrated_[BACK_RIGHT_MOTOR_CHIP_SELECT])
         << "Running without encoder calibration can cause serious harm, exiting";
 
-    // Get current wheel electical RPMs (don't account for pole pairs)
+    // Get current wheel electical RPMs (don't account for pole pairs). Values will be
+    // written on next iteration
```

Comment:
so to be clear, before we had motor.cpp readInt for the current values, ramp the targets based of the read values, and then calls writeInt. With the new changes, we read the values, and then also prepare writing the target velocities for the next iteration by ramping them with the read values. Is that correct and do you think u can add a comment describing the new behaviour i.e. smth along lines of "read motor values, ramp new control values based of read values, then set variables to hold the target velocities for the next iter etc."

Reply:
or just change `poll()` api desc.

Reply:
we assume that the motor has ramped to the speed we requested from the prev iteration as  we command higher velocities